src/webhooks/test/registry.test.ts

```diff
@@ -705,9 +785,18 @@ describe('ShopifyWebhooks.Registry.getTopics', () => {
   });
 
   it('adds two handlers and gets them', async () => {
-    ShopifyWebhooks.Registry.addHandler('PRODUCTS', {path: '/webhooks', webhookHandler: genericWebhookHandler});
-    ShopifyWebhooks.Registry.addHandler('PRODUCTS_CREATE', {path: '/webhooks', webhookHandler: genericWebhookHandler});
-    expect(ShopifyWebhooks.Registry.getTopics()).toStrictEqual(['PRODUCTS', 'PRODUCTS_CREATE']);
+    ShopifyWebhooks.Registry.addHandler('PRODUCTS', {
+      path: '/webhooks',
+      webhookHandler: genericWebhookHandler,
+    });
+    ShopifyWebhooks.Registry.addHandler('PRODUCTS_CREATE', {
+      path: '/webhooks',
+      webhookHandler: genericWebhookHandler,
+    });
+    expect(ShopifyWebhooks.Registry.getTopics()).toStrictEqual([
+      'PRODUCTS',
+      'PRODUCTS_CREATE',
+    ]);
```

Comment:
These tests have a massive amount of copy-paste in them, and are too verbose to really understand. I realize that most of the changes here are formatting, but there's a part of me that would almost like to see it re-factored to use proper functions; this way, the tests could read like a spec.

As an example, here's what two of these tests could look like:

```typescript
it('adds multiple handlers with duplicates', async () => {
  addHandler('PRODUCTS');
  addHandlers({
    PRODUCTS_CREATE: '/webhooks',
    PRODUCTS: '/newerpath'
  });
  assertRegistryKeysLength(2);
  assertRegistryKeys(['PRODUCTS_CREATE', 'PRODUCTS'])
  assertRegisteredPath('PRODUCTS', '/newerpath');
});

it('adds two handlers and gets them', async () => {
  addHandler('PRODUCTS');
  addHandler('PRODUCTS_CREATE');
  assertTopics([
    'PRODUCTS',
    'PRODUCTS_CREATE'
  ]);
});

// there's a way to type this, I just don't know it off the top of my head

function addHandlers(hash) {
  const finalHash = {};

  for(let key in hash) {
    if(hash.hasOwnProperty(key)) {
      finalHash[key] = {
        path: hash[key],
        webhookHandler: genericWebhookHandler
      }
    }
  }

  await ShopifyWebhooks.Registry.addHandlers(finalHash);
}

function addHandler(type: string) {
  await ShopifyWebhooks.Registry.addHandler(type, {
    path: '/webhooks',
    webhookHandler: genericWebhookHandler,
  });
}

function assertRegistryKeysLength(expected: number) {
  expect(Object.keys(ShopifyWebhooks.Registry.webhookRegistry)).toHaveLength(expected);
}

function assertRegisteredPath(type: string, expected: string) {
  expected(ShopifyWebHooks.Registry.webhookRegistry[type].path).toBe(expected);
}

function assertRegistryKeys(expected: string[]) {
  expected(Object.keys(ShopifyWebhooks.Registry.webhookRegistry).sort()).toEqual(expected.sort());
}

function assertTopics(expected: string[]) {
  expect(ShopifyWebhooks.Registry.getTopics()).toStrictEqual(expected);
}
```

Thoughts?